js_modules/dagit/packages/core/src/instance/BackfillTable.test.tsx

```diff
@@ -0,0 +1,37 @@
+import {MockedProvider} from '@apollo/client/testing';
+import {render, screen, waitFor} from '@testing-library/react';
+import * as React from 'react';
+
+import * as Alerting from '../app/CustomAlertProvider';
+import {TestProvider} from '../testing/TestProvider';
+
+import {BackfillTable} from './BackfillTable';
+import {
+  BackfillTableFragmentFailedErrorStatus,
+  BackfillTableFragmentFailedError,
+} from './BackfillTable.mocks';
+
+// This file must be mocked because Jest can't handle `import.meta.url`.
+jest.mock('../graph/asyncGraphLayout', () => ({}));
+
+describe('BackfillTable', () => {
+  it('allows you to click "Failed" backfills for error details', async () => {
+    jest.spyOn(Alerting, 'showCustomAlert');
+
+    render(
+      <TestProvider>
```

Comment:
Do we still need the full `TestProvider`? My hope was that we might not need it anymore for some/most/all cases that are using `MockedProvider` for Apollo needs.